benchmarks/benchmark.ml

```diff
@@ -72,12 +137,32 @@ let benchmarks =
           ; exec_bench Re.execp "execp" re cases
           ; exec_bench Re.exec_opt "exec_opt" re cases ]
       ) in
+  let http_benches =
+    let open Bench in
+    let open Http.Export in
+    let manual =
+      [ request, "no group" ; request_g, "group" ]
+      |> List.map ~f:(fun (re, name) ->
+          Test.create ~name (fun () -> read_all_http 0 re http_requests)
+        )
+      |> Test.create_group ~name:"manual" in
+    let many =
+      [ Test.create ~name:"no group" (fun () ->
+            ignore (Re.execp requests http_requests)
```

Comment:
This should have the same performances without or without groups (since the recent update to match_str). Do we have a bench that tests that ?